### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ WolkAbout C++ Connector library, and example are built from 'out' directory by i
 Example Usage
 -------------
 
-Create a device on WolkAbout IoT platform by importing manifest file `simple-example-manifest.json` located in `examples/simple/`
+Create a device on WolkAbout IoT platform using the Device Type 'Simple example'.
 This manifest fits `simple` example and demonstrates the sending of a temperature sensor reading.
```
Comment (djesic) [inline README.md]: point simple to application.cpp

File: examples/full_feature_set/README.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ WolkAbout C++ Connector library, and example are built from 'out' directory by i
 Example Usage
 -------------
 
-Create a device on WolkAbout IoT platform by importing manifest file `full-example-manifest.json` located in `examples/full_feature_set/`
+Create a device on WolkAbout IoT platform using the Device Type 'Full example'.
```
Comment (djesic) [inline examples/full_feature_set/README.md:51]: point to cpp

**Review by djesic — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (djesic) [reply README.md]: manifest to device type

**Review by djesic — CHANGES_REQUESTED** [review]

File: examples/full_feature_set/README.md (hunk 2 above)
Comment (ghost) [reply examples/full_feature_set/README.md:51]: Resolved, in line after this.

**Review by djesic — APPROVED** [review]

**Pull request merged**